ios/RNIterableAPI/Serialization.swift

```diff
@@ -143,8 +143,9 @@ extension IterableHtmlInAppContent {
         var dict = [AnyHashable: Any]()
         dict["type"] = type.rawValue
         dict["edgeInsets"] = edgeInsets.toDict()
-        dict["backgroundAlpha"] = backgroundAlpha
         dict["html"] = html
+        dict["shouldAnimate"] = shouldAnimate
+        dict["backgroundColor"] = backgroundColor ?? nil
```

Comment:
Instead of backgroundColor, our actual JSON hierarchy looks like this
content
--inappDisplaySettings
----shouldAnimate
----bgColor
------"hex":String, 
------"alpha": Float 
--html

I remember the idea of having a seperate getHtmlContent method was to provide just the html cosmetic contents for those who need it.. I think contents inside inapp display settings are unnecessary for them. They can always have extra information in getInAppMessage where they can get everything about the IterableInAppMessage
@vbabenkoru need your thoughts on it as well.

Reply:
Looks like this is an interface between RN and native, so it doesn't necessarily have to follow the same structure as the JSON payload; do we expose these properties in native SDKs?